jscomp/ml/translcore.ml

```diff
@@ -317,9 +317,9 @@ let primitives_table =
       ("%obj_field", Parrayrefu);
       ("%obj_set_field", Parraysetu);
       ("%obj_is_int", Pisint);
-      ("%raise", Praise Raise_regular);
-      ("%reraise", Praise Raise_reraise);
-      ("%raise_notrace", Praise Raise_notrace);
+      ("%raise", Praise);
+      ("%reraise", Praise);
+      ("%raise_notrace", Praise);
```

Comment:
We can also remove these two lines, but it'll be a breaking change.

Reply:
I only see those used in some tests, and `%raise_notrace` in Pervasives. I guess we can remove them there?

We'll need to clean up Pervasives anyway for v12.ras

Reply:
Should I remove them?

Reply:
I would say yes - do you agree @cristianoc?

Reply:
Sure